packages/openneuro-components/src/read-more/ReadMore.tsx

```diff
@@ -33,7 +33,7 @@ export const ReadMore = ({
 
           <section>
             {children}
-            <label htmlFor={id}>
+            <label htmlFor={id} className="expand-collapse">
```

Comment:
I guess `className` instead of `class` is because this is technically Javascript even though it looks a lot like HTML?

Reply:
Yes, it's a convention used by React because of the potential keyword conflict. Alternatives like Preact just use `class=""`.